addons/website_slides/controllers/main.py

```diff
@@ -670,6 +670,9 @@ def slide_channel_unsubscribe(self, channel_id):
     def slide_view(self, slide, **kwargs):
         if not slide.channel_id.can_access_from_current_website() or not slide.active:
             raise werkzeug.exceptions.NotFound()
+        # redirection to channel's homepage for category slides
+        if slide.is_category:
+            return werkzeug.utils.redirect("/slides/%s" % (slug(slide.channel_id)))
```

Comment:
Note that you could use "website_url" field of channel to hide a bit the low level details of url manipulation :)

Reply:
Hello @tde-banana-odoo I just pushed a new change using the "website_url", could you have a check? ^^ :pray: